aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -108,7 +108,13 @@ class PGCluster:
     _pg_roles: Dict[str, PGRole]
     _mangle: bool
 
-    def __init__(self, conn_info: Union[str, Dict[str, Any]], filtered_db: Optional[str] = None, mangle: bool = False):
+    def __init__(
+        self,
+        conn_info: Union[str, Dict[str, Any]],
+        filtered_db: Optional[str] = None,
+        filtered_roles: Optional[str] = None,
```

Comment:
I know that there's already a "filtered" concept that accepts an Optional[str], but I'd say this is a bad API design which mingles business objects with command line.

Suggestions: make filtered_roles a sequence of str, default to the empty tuple. You'll need fewer "ifs" to handle that.

In the cli parser, you can use an empty string as the default value, split on comma, and remove empty strings.

Reply:
Thanks, I've updated the code based on your feedback.